aws-credentials/tasks/main.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Create a dir
+  file:
+    path: /home/ubuntu/.aws/
+    state: directory
+    owner: ubuntu
+    group: ubuntu
+    mode: 0777
```

Comment:
More of a question for my own knowledge- why `0777` here?

Reply:
You can't get access a directory, unless its set to execute. We want to allow an attacked to have the opportunity to access the file.

Reply:
Good point though - for the generic role, maybe make a var with 700 perms or something along those lines, and we can change if required downstream with the var?